cmd/controller/main.go

```diff
@@ -201,7 +201,7 @@ func main() {
 		kubeClient, extClient, agonesClient, agonesInformerFactory)
 
 	rs = append(rs,
-		httpsServer, gsCounter, gsController, gsSetController, fleetController, faController, fasController, gasController, server)
+		httpsServer, gsCounter, gsCounter, gsController, gsSetController, fleetController, faController, fasController, gasController, server)
```

Comment:
Why do we need two gsCounter in one `rs` slice? Possibly we need to add a comment for that?
Also I think that comment in `perNodeCounter` Run() should be updated, because it is now blocking with mutex:
 https://github.com/GoogleCloudPlatform/agones/blob/b133e5280d2bec4bee98f78f59561cc2dc5a7171/pkg/gameservers/prenodecounter.go#L136

Reply:
whoops - that looks like a typo on my end. Let me remove that!

Reply:
Fixed. The comment was to say that the function doesn't block on `stop` (which many Run functions do) - I can change it to be more clear?

Reply:
Thanks for the clarification, I think no need to change comment, cause this file was not touched here. 🙇